File: No 09 - Redis Again/src/GameModel/Card.cs
using System;

namespace GameModel
{
    public class Card
    {
        public int CardID { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public byte Spell { get; set; }
        public byte Attack { get; set; }
        public byte Health { get; set; }
        public Hero Hero { get; set; }
    }   
}

File: No 12 - REST with Sequelize/src/hartstone/models/card.js
module.exports = (sequelize, DataTypes) => {

    let Card = sequelize.define('Card', {
        name: DataTypes.STRING,
        description: DataTypes.STRING,
        attack: DataTypes.INTEGER,
        health: DataTypes.INTEGER,
        spell: DataTypes.INTEGER
    });

    Card.associate = function (models) {
        Card.belongsTo(models.Hero, {
            onDelete: "CASCADE",
            foreignKey: 'heroId'
        });
    };

    return Card;
}